src/app/components/register/register.component.ts

```diff
@@ -17,7 +18,7 @@ import { User } from 'src/app/models/user';
 
 export class RegisterComponent implements OnInit {
 
-	batches: Batch[] = [];
+	batches: Batch;
```

Comment:
why was this changed to one batch if it's later assigned to "data" which seems to be an array of batches